test/spspSpec.js

```diff
@@ -118,11 +118,11 @@ describe('SPSP Module', function () {
         .get('/spsp')
         .reply(200, spspResponse)
 
-      const payment = yield SPSP.quoteDestination(this.plugin, 'alice@example.com', '10')
+      const payment = yield SPSP.quoteSource(this.plugin, 'alice@example.com', '12')
       assert.deepEqual(payment, {
         destinationAccount: "example.alice",
         connectorAccount: "example.connie",
-        sourceAmount: "10",
+        sourceAmount: "12",
```

Comment:
I got that, I'm just suggesting that in both cases you make the sourceAmount 12 and the destinationAmount 10 (or vice versa)